### ISSUE TITLE
ReconnectingClientFactory isn't documented in the HOWTOs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8339]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| spiv reported|
|-|-|
|Trac ID|trac#458|
|Type|defect|
|Created|2004-01-03 09:53:32Z|

<details><summary>Searchable metadata</summary>

```
trac-id__458 458
type__defect defect
reporter__spiv spiv
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1073123612000000 1073123612000000
changetime__1470831656201540 1470831656201540
version__ 
owner__ 
cc__spiv cc__itamarst cc__hypatia
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
clients.xhtml mentions how to write a simple ClientFactory -- but never points
out that ReconnectingClientFactory is already written, and it's probably exactly
what a significant proportion of client authors are looking for.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Added example of EchoClientFactory inheriting from ReconnectingClientFactory
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset f5426e42fce2794f7e32b21a3c7f130d3e5602ff
```
#!CommitTicketReference repository="" revision="f5426e42fce2794f7e32b21a3c7f130d3e5602ff"
Merge pull request #458 from twisted/3460-session-uid

Author: mthuurne, glyph

Reviewer: runciter

Fixes: #3460

Generate Session.uid more securely and simply.
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

In changeset 85c513264eba7470e50baebd723cfc2372602be3
```
#!CommitTicketReference repository="" revision="85c513264eba7470e50baebd723cfc2372602be3"
Merge pull request #458 from twisted/3460-session-uid

Author: mthuurne, glyph

Reviewer: runciter

Fixes: #3460

Generate Session.uid more securely and simply.
```
